.github/workflows/prebuild.yml

```diff
@@ -21,9 +21,9 @@ jobs:
         run: yarn --frozen-lockfile --ignore-scripts
       - name: fetch libcore
         run: yarn preinstall
-      - name: config yarn
```

Comment:
why remove this ?

Reply:
`node-gyp` is expecting to have VS 2017 to compile code, but only VS 2019 was available on the `windows-latest` image, so this step was needed to tell it where to find the compiler.

Now that the image has been changed to `windows-2016`, `node-gyp` is able to find VS 2017 on its own, and this step isn't needed anymore.